client/components/payment-activity/payment-activity-data.tsx

```diff
@@ -208,11 +209,11 @@ const PaymentActivityDataComponent: React.FC< Props > = ( {
 							searchTermsForViewReportLink.dispute
 						),
 					} ) }
-					tracksSource="disputes"
 					isLoading={ isLoading }
 				/>
 				<PaymentDataTile
 					id="wcpay-payment-data-highlights__fees"
+					tracksSource="fees"
```

Comment:
I didn't realise that this tile is not clickable like the others. So it doesn't really need a `tracksSource`. TBC

Reply:
Marking PR as blocked, discussing internally.

May back out of this PR since the main benefit is adding fees source, without that it's not a good idea to make the prop mandatory.